R/stan_module.R

```diff
@@ -212,3 +213,37 @@ remove_blank_strings <- function(x) {
     return(x[!x == ""])
 }
 
+
+#' Read entire file as a single string
+#'
+#' Simple utility function to read in a file as 1
+#' continous string
+#'
+#' @param filename Location of file to read in
+#'
+read_file <- function(filename) {
+    paste0(readLines(filename), collapse = "\n")
```

Comment:
I am thinking that maybe changing line in the end of the character string could be helpful for later `paste0(paste0(readLines(filename), collapse = "\n"), "\n")`. eg when we will add "}", we want them to be in an empty line at the end.

Reply:
I think that would make more sense to go into the `as.character()` method no ?  As in (though unlikely) there is a chance this function could get used for other purposes where it doesn't make sense to have that trailing newline.

Reply:
Your suggestion sounds more reasonable way to proceed.